tools/benchmark_tool/openvino/tools/benchmark/main.py

```diff
@@ -216,42 +218,42 @@ def set_throughput_streams():
             next_step()
 
             start_time = datetime.utcnow()
-            exe_network = benchmark.load_network(args.path_to_model)
+            exe_network = benchmark.ie.compile_model(args.path_to_model)
             duration_ms = f"{(datetime.utcnow() - start_time).total_seconds() * 1000:.2f}"
-            logger.info(f"Load network took {duration_ms} ms")
+            logger.info(f"Compile model took {duration_ms} ms")
             if statistics:
                 statistics.add_parameters(StatisticsReport.Category.EXECUTION_RESULTS,
                                           [
-                                              ('load network time (ms)', duration_ms)
+                                              ('compile model time (ms)', duration_ms)
                                           ])
-            app_inputs_info, _ = get_inputs_info(args.shape, args.layout, args.batch_size, args.input_scale, args.input_mean, exe_network.input_info)
+            app_inputs_info, _ = get_inputs_info(args.shape, args.layout, args.batch_size, args.input_scale, args.input_mean, exe_network.get_runtime_function().get_parameters())
             if batch_size == 0:
                 batch_size = 1
         elif not is_network_compiled:
             # --------------------- 4. Read the Intermediate Representation of the network -----------------------------
             next_step()
 
             start_time = datetime.utcnow()
-            ie_network = benchmark.read_network(args.path_to_model)
-            topology_name = ie_network.name
+            network = benchmark.read_model(args.path_to_model)
```

Comment:
small comment: it's better to rename it to function because `read_model` return Function object